tests/pytests/common.py

```diff
@@ -114,13 +114,14 @@ def toSortedFlatList(res):
         return py2sorted(finalList)
     return [res]
 
-def assertInfoField(env, idx, field, expected, delta=None):
-    if not env.isCluster():
-        d = index_info(env, idx)
-        if delta is None:
-            env.assertEqual(d[field], expected)
-        else:
-            env.assertAlmostEqual(float(d[field]), float(expected), delta=delta)
+def assertInfoField(env, idx, field, expected, delta=None, test_inside_cluster=False):
+    if env.isCluster() and not test_inside_cluster:
+        return
```

Comment:
Have you tried to remove this condition now that we improved the support of info fields in cluster mode? If we cannot, consider making the default value of `test_inside_cluster` to `True` so we need to explicitly opt out of the verification when the replies are not aligned.

Reply:
Will give it a try

Reply:
I would also rename it to `test_on_cluster`, `test_on_cluster_mode` or `skip_on_cluster` (and flipping the logic)

Reply:
Looks like the tests pass without the filtering.
Will remove the if and the parameter